include/lemlib/devices/gyro/gyro.hpp

```diff
@@ -42,23 +44,23 @@ class Gyro {
          *
          * @return float heading, in radians, locked from -pi to +pi
          */
-        virtual float getHeading() = 0;
+        virtual Angle getHeading() = 0;
         /**
          * @brief Get the rotation of the gyro
          *
          * @note 0 is in the positive x direction, and heading increases counterclockwise
          *
          * @return float rotation, in radians
```

Comment:
```suggestion
         * @return Angle rotation
```
Update doxygen comment